styleguide/source/_patterns/04-templates/three-column.twig

```diff
@@ -1,42 +1,23 @@
-{% include '@atoms/skip-nav.twig' %}
+<div class="ama__layout--three-column">
+  {# Keeping tabindex for accessibility purposes #}
+  <article class="ama__page-content" role="article">
+    {% block pageContent %}
+      {% set placeholder = three_column.page_content.placeholder %}
+      {% include '@base/placeholder.twig' %}
+    {% endblock %}
+  </article>
 
-<div class="container">
-  {% block ribbon %}
-    {% set placeholder = three_column.ribbon.placeholder %}
-    {% include '@base/placeholder.twig' %}
-  {% endblock %}
+  <aside id="sidebar-primary" class="ama__sidebar-primary" role="complementary">
+    {% block sidebarPrimary %}
+      {% set placeholder = three_column.sidebar_primary.placeholder %}
+      {% include '@base/placeholder.twig' %}
+    {% endblock %}
+  </aside>
 
-  {% block header %}
-    {% set placeholder = three_column.header.placeholder %}
-    {% include '@base/placeholder.twig' %}
-  {% endblock %}
-
-  <main id="main-content" class="ama__layout--three-column" tabindex="-1" role="main">
-    {# Keeping tabindex for accessibility purposes #}
-    <article class="ama__page-content" role="article">
-      {% block pageContent %}
-        {% set placeholder = three_column.page_content.placeholder %}
-        {% include '@base/placeholder.twig' %}
-      {% endblock %}
-    </article>
-
-    <aside id="sidebar-primary" class="ama__sidebar-primary" role="complementary">
-      {% block sidebarPrimary %}
-        {% set placeholder = three_column.sidebar_primary.placeholder %}
-        {% include '@base/placeholder.twig' %}
-      {% endblock %}
-    </aside>
-
-    <aside id="sidebar-secondary" class="ama__sidebar-secondary" role="complementary">
-      {% block sidebarSecondary %}
-        {% set placeholder = three_column.sidebar_secondary.placeholder %}
-        {% include '@base/placeholder.twig' %}
-      {% endblock %}
-    </aside>
-  </main>
-
-  {% block footer %}
-    {% set placeholder = three_column.footer.placeholder %}
-    {% include '@base/placeholder.twig' %}
-  {% endblock %}
-</div>
+  <aside id="sidebar-secondary" class="ama__sidebar-secondary" role="complementary">
```

Comment:
Since "secondary" is a modifier of "sidebar" the class should be `ama__sidebar--secondary`